tests/setup_testsubproject.cmake

```diff
@@ -1,140 +1,17 @@
-## ---------------------------------------------------------------------
-##
-## Copyright (C) 2013 by the deal.II authors
-##
-## This file is part of the deal.II library.
-##
-## The deal.II library is free software; you can use it, redistribute
-## it, and/or modify it under the terms of the GNU Lesser General
-## Public License as published by the Free Software Foundation; either
-## version 2.1 of the License, or (at your option) any later version.
-## The full text of the license can be found in the file LICENSE at
-## the top level of the deal.II distribution.
-##
-## ---------------------------------------------------------------------
+FIND_PACKAGE(deal.II 8.0 REQUIRED HINTS ${DEAL_II_DIR})
```

Comment:
Am 27. Mar 2015, 19:37 schrieb Wolfgang Bangerth <notifications@github.com>:

>> @@ -1,140 +1,17 @@
>> -## ---------------------------------------------------------------------
>> -##
>> -## Copyright (C) 2013 by the deal.II authors
>> -##
>> -## This file is part of the deal.II library.
>> -##
>> -## The deal.II library is free software; you can use it, redistribute
>> -## it, and/or modify it under the terms of the GNU Lesser General
>> -## Public License as published by the Free Software Foundation; either
>> -## version 2.1 of the License, or (at your option) any later version.
>> -## The full text of the license can be found in the file LICENSE at
>> -## the top level of the deal.II distribution.
>> -##
>> -## ---------------------------------------------------------------------
>> +FIND_PACKAGE(deal.II 8.0 REQUIRED HINTS ${DEAL_II_DIR})
>
> Did you remove the copyright notice on purpose?

No, this is a minor refactoring accident.